cws-test/pom.xml

```diff
@@ -107,7 +107,14 @@
 			<artifactId>h2</artifactId>
 			<scope>test</scope>
 		</dependency>
-		
+
+		<dependency>
+			<groupId>io.github.bonigarcia</groupId>
```

Comment:
Is this the official source for this library?

Reply:
Yes, according to Selenium documentation